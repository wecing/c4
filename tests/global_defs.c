extern int printf(const char *s, ...);

extern const char *const names[];

const char *const names[] = {
  "aaa", "bbb", "ccc", "ddd",
};

int n;

int f() {
  return n;
}

int n = 5;
int n;

int main() {
  printf("n = %d\n", n); // 5
  printf("f() = %d\n", f()); // 5
  return 0;
}
