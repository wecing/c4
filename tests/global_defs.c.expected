n = 5
f() = 5
